internal/json/string.go

```diff
@@ -123,6 +125,20 @@ func AppendBytes(dst, s []byte) []byte {
 	return append(dst, '"')
 }
 
+// AppendHex encodes the input bytes to a hex string and appends
+// the encoded string to the input byte slice.
+//
+// The operation loops though each byte and encodes it as hex using
+// the hex lookup table.
+func AppendHex(dst, s []byte) []byte {
+	dst = append(dst, '"')
+	for _, v := range s {
+		dst = append(dst, hex[v>>4], hex[v&0x0f])
+	}
+	dst = append(dst, s...)
```

Comment:
I don't understand this line.

Reply:
Stupid copy-paste mistake. Fixing it up with the tests now.